libvirt-formula/libvirt/defaults/sockets.yaml

```diff
@@ -0,0 +1,2 @@
+---
+tcp: true
```

Comment:
I would recommend add one more setting:
tls : false

there is one more possible socket libvirtd-tls.socket. That is using SSL certs, maybe we will switch into it in the future.

Reply:
Without it being specified, it's not touched and left unconfigured. On clean systems it should not be enabled to begin with - but it can be set in the pillar if one wants it to be explicitly disabled.

Reply:
yes please the idea is that is future we can test it via simple switch it to true.

Reply:
This is how it already is.